src/subdomains/core/sell-crypto/route/sell.entity.ts

```diff
@@ -37,6 +37,6 @@ export class Sell extends DepositRoute {
   // --- ENTITY METHODS --- //
 
   get userData(): UserData {
-    return this.user.userData;
+    return this.user?.userData;
```

Comment:
Why do we need that? 🤔 
`userData` is nullable false in `user` or?

I think I would like to have exception for a missing relation in such situations to find it 😅